src/citra_qt/sbs_3d_ui.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright 2014 Citra Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <algorithm>
+#include <glad/glad.h>
+#define QT_NO_OPENGL
+#include <QMessageBox>
+#include <QtWidgets>
+#include "citra_qt/sbs_3d_ui.h"
+#include "core/settings.h"
+
+void MoveDialogToLeftEye(QWidget* dialog, QWidget* parent) {
+    // move dialog to centre of left eye in SBS 3D mode
+    if (Settings::values.toggle_3d) {
```

Comment:
This can be converted into a guard clause:

```suggestion
    if (!Settings::values.toggle_3d) {
        return;
    }
```

Reply:
Less indentation so code becomes easier to read. Also it's part of the style Citra uses.